tests/models/test_gaussian_election_model.py

```diff
@@ -1,23 +1,6 @@
 from elexmodel.models import GaussianElectionModel
 
 
-def test_instantiation():
```

Comment:
curious what is happening here

Reply:
This check was one I wrote from the first iteration when we were passsing beta and winsorize into the model directly through fit. When we decided to pass them through model settings instead, it was no longer needed